### DISCUSSION TITLE
npx prisma generate is no working on server

### DISCUSSION THREAD

**Discussion by ashwin31** [prisma/prisma#23147] [Q&A] [answered]
### Question

how to fix prisma generate hen i get 403 error.

i guess my server ip address got blocked by binaries.prisma.sh

### How to reproduce (optional)

2. Change '....'
3. Run 'npx prisma generate'
$ npx prisma generate
> Downloading Prisma engines for Node-API for debian-openssl-1.1.x [                    ] 0%Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/23fdc5965b1e05fc54e5f26ed3de66776b93de64/debian-openssl-1.1.x/libquery_engine.so.node.sha256 - 403 Forbidden

If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment, marked as answer]
Hey @ashwin31 👋 

Can you try changing your network and check?
Are you using any kind of firewall that is preventing the binaries from getting installed?

**Reply by ashwin31** [reply]
No firewalls, everything installing but not this command.

**Comment by janpio** [comment]
Can you open and download https://binaries.prisma.sh/all_commits/23fdc5965b1e05fc54e5f26ed3de66776b93de64/debian-openssl-1.1.x/libquery_engine.so.node.sha256 manually in your browser?

**Reply by ashwin31** [reply]
yes, and the command started working now. definately some network issue with binaries.prisma.sh 

Thank you for helping me.

**Comment by MatrixManAtYrService** [comment]
I'm also getting `Forbidden` from binaries.prisma.sh

```
circleci@ip-10-0-188-193:~/project/.localdev/astro/apps/authz$ yarn build
yarn run v1.22.19
$ yarn prisma:gen && tsoa spec-and-routes && tsc && yarn format-prettier
$ prisma generate
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [                    ] 0%Error: Forbidden https://binaries.prisma.sh/all_commits/efdf9b1183dddfd4258cd181a72125755215ab7b/debian-openssl-3.0.x/introspection-engine.gz
error Command failed with exit code 1.
```

Only in CI though, the command works OK locally.  Is this perhaps a rate limiting measure?

**Reply by felipesilva-plank** [reply]
any solutions for this? I'm getting this error on the CI as well

**Reply by z4122** [reply]
> any solutions for this? I'm getting this error on the CI as well

Me too, I got this Failed to fetch the engine file at https://binaries.prisma.sh/all_commits/173f8d54f8d52e692c7e27e72a88314ec7aeff60/linux-musl-openssl-3.0.x/schema-engine.gz - 403 Forbidden. 
But it can work several hours ago.

**Reply by pgerlich** [reply]
Same, also getting this as of very recently in CI on Github actions only. Works fine locally. @sorenbs any ideas? Github issue?

**Reply by saroj-mhj** [reply]
Im having same issue.
https://binaries.prisma.sh/all_commits/8047c96bbd92db98a2abc7c9323ce77c02c89dbc/linux-musl-arm64-openssl-3.0.x/schema-engine.gz.sha256

**Comment by basatmaqsood** [comment]
I had same issue and I used VPN and it worked for me

**Comment by videvian** [comment]
Just got the same issue, this is returning 403 https://binaries.prisma.sh/all_commits/f40f79ec31188888a2e33acda0ecc8fd10a853a9/debian-openssl-3.0.x/schema-engine.gz.sha256, and it seems like `debian-openssl-3.0.x` is not on https://binaries.prisma.sh/.

```shell
Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/f40f79ec31188888a2e33acda0ecc8fd10a853a9/debian-openssl-3.0.x/schema-engine.gz.sha256 - 403 Forbidden

If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1
```

**Reply by junedkhatri31** [reply]
I'm also facing this issue.

**Reply by z4122** [reply]
Me too

**Reply by junedkhatri31** [reply]
Issue with prisma S3 only

**Reply by charlesduvert** [reply]
Same issue ...

**Reply by junedkhatri31** [reply]
@prisma Please don't do deplyments on Fridays

**Reply by enzo-base-labs** [reply]
Same issue ...

**Reply by doanhtu07** [reply]
Yo what happened? It was literally working a few minutes ago, and now it's not :)))

**Reply by JonBernardHealia** [reply]
same

**Reply by mwajnman** [reply]
same

**Reply by nurul3101** [reply]
We are investigating this right now. I'll inform you as soon as it's resolved.

**Reply by gus-costa** [reply]
+1

**Reply by KhudhurMD** [reply]
+1

**Reply by bozkurtozdemir95** [reply]
same +

**Reply by ErickGrCosta** [reply]
Same here!

**Reply by ThLeee** [reply]
same +1 ☕

**Comment by xhackermanx** [comment]
![755913669534312](https://github.com/user-attachments/assets/c949c6fd-e76a-475e-86b2-84979971ae57)


```
npx prisma generate
> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/8047c96bbd92db98a2abc7c9323ce77c02c89dbc/windows/query_engine.dll.node.gz.sha256 - 403 Forbidden

If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1
```

**Comment by felipesilva-plank** [comment]
after setting `PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1` in my CI pipelines env, I now get:

```
Error: Failed to fetch the engine file at https://binaries.prisma.sh/all_commits/605197351a3c8bdd595af2d2a9bc3025bca48ea2/debian-openssl-3.0.x/libquery_engine.so.node.gz - 403 Forbidden
```

It might be something on Prisma servers. 
@nurul3101

**Reply by nurul3101** [reply]
We are looking into this right now

**Reply by peter-citeright** [reply]
I guess now is a good time for a coffee break..

**Reply by dom-cr** [reply]
Guess it's time to switch to Oracle

**Comment by charlesduvert** [comment]
Same error on :

Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/8047c96bbd92db98a2abc7c9323ce77c02c89dbc/linux-musl-arm64-openssl-3.0.x/libquery_engine.so.node.gz.sha256 - 403 Forbidden

If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1

@nurul3101

**Reply by nurul3101** [reply]
Yes, we are working on fixing it as we speak.

**Reply by bryanmaraujo544** [reply]
@nurul3101 Is it being updated at Prisma status page?

**Reply by nurul3101** [reply]
Yes, we updated the status page: https://www.prisma-status.com/

**Comment by George-Miao** [comment]
Lol guess prisma is nice enough to give everyone a coffee break... CI is completely unusable now

**Comment by jcyuan** [comment]
#27701 

same... ☕

**Comment by noalmeida** [comment]
the same problem started and recently, until a few days ago it was normal. The same issue started recently, and until a few days ago, it was normal. It seems to be happening across the binary organization, and the Prisma team is already active.

**Comment by luis-sant1** [comment]
Same error
`> Downloading Prisma engines for Node-API for windows [              ] 0%Error: Failed to fetch the engine file at https://binaries.prisma.sh/all_commits/8047c96bbd92db98a2abc7c9323ce77c02c89dbc/windows/schema-engine.exe.gz - 403 Forbidden`
Even using PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1

*EDIT: Now is working!

**Comment by RonaldGuilhermePDS** [comment]
The same

Example: PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING=1
If you need to ignore this error (e.g. in an offline environment), set the PRISMA_ENGINES_CHECKSUM_IGNORE_MISSING environment variable to a truthy value.
Error: Failed to fetch sha256 checksum at https://binaries.prisma.sh/all_commits/3cff47a7f5d65c3ea74883f1d736e41d68ce91ed/linux-musl-openssl-3.0.x/schema-engine.sha256 - 403 Forbidden
 ---> Running in 805439bd3f92

**Comment by berlinbra** [comment]
Is there an update on this issue? Or an update on the known cause?
I am experiencing this error when trying to set up prisma on a pythonanwyere server
